pkg/compiler/compiler_func_ns_test.go

```diff
@@ -78,4 +80,72 @@ func TestFunctionNSCall(t *testing.T) {
 
 		So(err, ShouldNotBeNil)
 	})
+
+	Convey("Should use keywords", t, func() {
+		c := compiler.New()
+
+		keywords := []string{
+			"And",
+			"Or",
+			"For",
+			"Return",
+			"Distinct",
+			"Filter",
+			"Sort",
+			"Limit",
+			"Let",
+			"Collect",
+			"SortDirection",
+			"None",
+			"Null",
+			"BooleanLiteral",
+			"Use",
+			"Into",
+			"Keep",
+			"With",
+			"Count",
+			"All",
+			"Any",
+			"Aggregate",
+			"Like",
+			"Not",
+			"In",
+		}
+
+		for _, kw := range keywords {
+			segment := strings.ToUpper(kw)
+			err := c.Namespace("T").Namespace(segment).RegisterFunction("TEST", func(ctx context.Context, args ...core.Value) (core.Value, error) {
```

Comment:
maybe we should add package `internal/testutil` that contains stub like 
```go
func True(ctx context.Context, args ...core.Value) (core.Value, error) {
    return values.True, nil
}
```
to not rewriting it every time?

also line is too long :)

Reply:
The function itself is not important here. It does not matter what it returns.